tools/benchmark/src/mocha/memoryTestRunner.ts

```diff
@@ -296,6 +315,32 @@ export function benchmarkMemory(testObject: IMemoryTestObject): Test {
 						heapUsedStats.marginOfErrorPercent > args.maxRelativeMarginOfError
 					);
 
+					if (baselineMemoryUsage >= 0 && allowedDeviationBytes >= 0) {
+						// Compare the average heap used to the baseline memory usage
+						const avgHeapUsed = heapUsedStats.arithmeticMean;
+						const lowerBound = baselineMemoryUsage - args.allowedDeviationBytes;
+						const upperBound = baselineMemoryUsage + args.allowedDeviationBytes;
+						const tolerance = (args.allowedDeviationBytes / baselineMemoryUsage) * 100;
```

Comment:
This can result in a division by 0. I know I said before that keeping the tolerance as percentage seemed fine, to avoid the conversion; now that we receive it in bytes, I think similar reasoning applies to now keep it in bytes, i.e. just use `allowedDeviationBytes` 😄 .